README.md

```diff
@@ -9,6 +9,10 @@ to their data as a TensorFlow graph. This is important as the user can then
 incorporate the exported TensorFlow graph into their serving model, thus
 avoiding skew between the served model and the training data.
 
+# tf.Transform Samples
```

Comment:
Perhaps discuss with @KesterTong regarding what the best place for this is?

Should it be here or in the Getting Started Section or the more examples section?

Reply:
I am trying to optimize for quick and easy discovery for samples @katsiapis @KesterTong Thoughts?

Reply:
Those samples are in the context of using Google Cloud ML Engine and also demonstrate how to use tf.Transform with Google Cloud Dataflow, while the samples in this repo give more detail on how to use tf.Transform in general.

Therefore I think that in the intro we should add a new paragraph (not a new section) here saying "tf.Transform works seamlessly with Google CloudML Engine.  See \*examples\*  for examples of using tf.Transform with Google CloudML Engine"

And in the "Getting Started" section I would add sentence along the lines of "For examples of how to run tf.Transform on Google Cloud Dataflow, see \*examples\*.  These examples also demonstrate training with Google CloudML Engine.